src/actions/FirstOpenActions.tsx

```diff
@@ -42,9 +52,11 @@ const readFirstOpenInfoFromDisk = async (): Promise<FirstOpenInfo> => {
       firstOpenText = await firstOpenDisklet.getText(FIRST_OPEN)
       firstOpenInfo = asFirstOpenInfo(JSON.parse(firstOpenText))
       firstOpenInfo.isFirstOpen = 'false'
+      firstOpenInfo.appleAdsAttribution = await getAppleAdsAttribution()
```

Comment:
IF we load this on every boot, why even save it? Please remove this from the file.

One way to do this is to separate the types for the file and the in-memory data type:

```ts
interface FirstOpenInfo {
  isFirstOpen: boolean,
  deviceId: string,
  firstOpenEpoch: number,
  countryCode?: string,
  appleAdsAttribution?: AppleAdsAttribution
}

type FirstOpenInfoFile = Omit<FirstOpenInfo, 'appleAdsAttribution'>
const asFirstOpenInfoFile = asObject<FirstOpenInfoFile>({
  ...
})
```

It might make sense to create a whole new place to put stuff like this, instead of jamming it into this grab-bag file, but I see that that's a bigger change.